w-books/src/assets/styles.css

```diff
@@ -0,0 +1,35 @@
+.Home {
```

Comment:
Take care to create styles only in components folders.
```
Components
        |  Header
        |          | . index.js
        | .        | . styles.js
screens
   | Home
   |        |  index.js
   |        |  layout.js
   |        |  styles.js

   | OtherScreen
   |        |  index.js
   |        |  layout.js
   |        |  styles.js
```
each component have a style file